docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,79 @@
+How to configure boot parameters on Ubuntu Core 
+================================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+This guide depends on a running Ubuntu Core instance with the real-time kernel. 
+This should be on a physical device rather than a virtual machine.
+While a virtual machine can be used to follow the steps in this guide, it will not provide the benefits of a real-time kernel.
+
+Ubuntu Core doesn't have the :code:`/etc/default/grub` file mapped for the kernel parameters, neither the `update-grub`_ utility.
+On Ubuntu Core, the kernel parameters can be modified statically or dynamically.
+Static configurations should be added during the image build; refer to :doc:`../how-to/uc-image-creation`.
+Dynamic configurations can be set `using snap options`_ on a running system.
+This guide focuses on the dynamic configuration method, suitable for iteratively tuning the operating system.
+
+There are two snap options for setting the kernel parameters:
+
+- `system.kernel.cmdline-append`_
+- `system.kernel.dangerous-cmdline-append`_
+
+The first one is used to append permitted kernel boot parameters that are verified against an `allow list`_ in the `gadget snap`_. 
+The second one is used to append any other kernel boot parameters, considered as not analyzed by the gadget snap.
+
+When using the reference gadget snaps (`pc-gadget`_ and `pi-gadget`_), you need to use the `system.kernel.dangerous-cmdline-append` option, as the allow list is not present in them.
+
+When creating a custom `gadget snap`_ for a real-time Ubuntu Core system, it is advisable to add an allow list with the relevant parameters.
+
+The kernel parameters in the snap configurations exactly match the `kernel parameters`_ that would be passed to the kernel on the grub command line, the :code:`GRUB_CMDLINE_LINUX_DEFAULT` on :code:`/etc/default/grub` file. 
+Parameters are passed as a single string in the :code:`key=value` format, with each parameter separated by a space.
+
+In the case of :doc:`real-time kernel parameters <../reference/kernel-boot-parameters>`, the most interesting parameters are `nohz`_, `nohz_full`_ and `irqaffinity`_.
+The :code:`nohz` is used to enable/disable dynamic ticks (possible values are `on` or `off`). 
+The :code:`nohz_full` receives a `cpu list`_ specifying which CPUs will have dynamic ticks disabled. 
+The `irqaffinity` is used to set the affinity of the IRQs to the CPUs, in other words: which CPUs will handle the IRQs.
```

Comment:
I don't know if this is possible but it would be great if by hovering an accronym I would get it's definition. I had to search 'IRQs' on search bar to find it.

Reply:
It might be possible, but a tooltip will be hard to use on touch screen devices. I've expanded the acronym.

Reply:
This is a little late but there is a Sphinx directive (`:abbr:`) for this exact use.
https://github.com/canonical/sphinx-docs-starter-pack/blob/main/doc-cheat-sheet.rst?plain=1#L243

But unfortunately as Farshid mentioned, this doesn't work for mobile devices / touch screens.

For Kernel docs we use a Glossary to help address this so we can direct users to a complete list of terms (including acronyms + their definitions) so we don't have to worry about expanding the acronym on each occurrence.